source/templates/code_signing.slim

```diff
@@ -24,28 +24,33 @@
 			h2.label.expose[ng-if="provProfileCtrl.provProfiles.length > 0"]
 				span.long-version == data[:strings][:code_signing][:prov_profile][:header][:expose_for_pull_requests]
 				span.short-version == data[:strings][:code_signing][:prov_profile][:header][:expose_for_pr]
+			.actions[ng-class="{'hidden': provProfileCtrl.provProfiles.length <= 1}" floating-menu toggle="provProfileCtrl.actionsForAllProvisioningProfiles.isMenuVisible"]
+				button.dropdown-toggle[ng-if="provProfileCtrl.provProfiles.length > 1" ng-click="provProfileCtrl.actionsForAllProvisioningProfiles.isMenuVisible = !provProfileCtrl.actionsForAllProvisioningProfiles.isMenuVisible" ng-disabled="provProfileCtrl.deleteAllProvProfilesProgress.isInProgress"] = svg("three-horizontal-dots")
```

Comment:
If there are no prov profiles, dropdown toggle will not be present, thus the dropdown will never be opened, thus actions will be an empty div? That's not cool.